1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,107 @@
 // Write your code here
+// level 1 challenge
+
+// assign my src codes
+const picturesSources = [
+    "/1-mandatory/3-slideshow/posters/1.jpeg",
+  "/1-mandatory/3-slideshow/posters/2.jpeg",
+  "/1-mandatory/3-slideshow/posters/3.jpeg",
+  "/1-mandatory/3-slideshow/posters/4.jpeg",
+  "/1-mandatory/3-slideshow/posters/5.jpeg",
+  "/1-mandatory/3-slideshow/posters/6.jpeg",
+  "/1-mandatory/3-slideshow/posters/7.jpeg",
+  "/1-mandatory/3-slideshow/posters/8.jpeg",
+  "/1-mandatory/3-slideshow/posters/9.jpeg",
+  "/1-mandatory/3-slideshow/posters/10.jpeg",
+  "/1-mandatory/3-slideshow/posters/11.jpeg",
+  "/1-mandatory/3-slideshow/posters/12.jpeg",
+  "/1-mandatory/3-slideshow/posters/13.jpeg",
+  "/1-mandatory/3-slideshow/posters/14.jpeg",
+  "/1-mandatory/3-slideshow/posters/15.jpeg",
+  "/1-mandatory/3-slideshow/posters/16.jpeg",
+  "/1-mandatory/3-slideshow/posters/17.jpeg",
+  "/1-mandatory/3-slideshow/posters/18.jpeg",
+  "/1-mandatory/3-slideshow/posters/19.jpeg",
+  "/1-mandatory/3-slideshow/posters/20.jpeg",
+  "/1-mandatory/3-slideshow/posters/21.jpeg",
+  "/1-mandatory/3-slideshow/posters/22.jpeg",
```

Comment:
You should reference your marvel posters using a relative path, for example: `./posters/1.jpeg`. Otherwise, browser fails to load resource if you open `index.html` directly.

Reply:
changed them, thank you